package.json

```
@@ -13,11 +14,10 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/aichner/React-MDB-Template.git"
+    "url": "https://github.com/snek-at/front.git"
```

Can't you use this fancy method here too?

```suggestion
    "url": "git+https://github.com/snek-at/front.git"
```